src/components/TextInputFocusable/index.native.js

```diff
@@ -48,6 +57,7 @@ class TextInputFocusable extends React.Component {
                 ref={el => this.textInput = el}
                 maxHeight={116}
                 rejectResponderTermination={false}
+                editable={!this.props.isDisabled}
```

Comment:
resolving `editable` to false would make the field read only it will also dismiss focus and prevent focusing the field while it's not editable

The "big" issue on Android still happens, but since the view is not unmounted it doesn't crash the app, though the Keyboard will pop out 

It seems the issue doesn't occur all the time though, Today when I started it didn't happen for a while, but then suddenly it started to happen again. It's like tapping in the area on the left that can be swiped to open the drawer somehow triggers this behavior